frontend/src/scenes/session-recordings/player/controller/PlayerController.tsx

```diff
@@ -48,7 +56,7 @@ function PlayPauseButton(): JSX.Element {
             ) : endReached ? (
                 <IconRewindPlay className="text-3xl" />
             ) : (
-                <IconPlay className="text-3xl" />
+                <GhostIcon className="LemonIcon text-3xl" /> //TODO: After halloween, reset to <IconPlay className="text-3xl" />
```

Comment:
we could use current date to choose ghost or not so we don't have to schedule things

or a feature flag so it's easy to flip/test

importantly, can it scream when you click it?

Reply:
actually, i'm 90% certain you can schedule feature flag changes so...

Reply:
The current timestamp might be better to account for time zones. I've done easter eggs in the past, and people were confused when April Fools stuff started showing up in US East the night before, because we were using UTC. We can just do some calcs and only show up once it's Halloween on PST, but then we lose Europe, etc. etc. etc.